### ISSUE TITLE
Travis should build on Raspbian

### ISSUE THREAD

**Issue by martukas** [RespiraWorks/VentilatorSoftware#3]
Here are some potential leads on how to do this:
https://docs.travis-ci.com/user/multi-cpu-architectures
http://blog.kragniz.eu/raspbian-on-travis-ci/

**Comment by twslankard** [comment]
New PR: https://github.com/RespiraWorks/VentilatorSoftware/pull/new/issue_13

**Comment by hhff** [comment]
@twslankard I think we can close this once #67 is merged, right?

**Comment by twslankard** [comment]
Yeah, that sounds fine to me.
